lib/simulation/openfisca/mapping/index.js

```diff
@@ -158,9 +158,9 @@ var mapIndividus = exports.mapIndividus = function (situation) {
     });
 
     return _.map(individus, function(individu) {
-        if (individu.role === 'demandeur') {
-            _.extend(individu, situation.patrimoine);
-
+        if (individu.role === 'demandeur' && situation.patrimoine) {
+            // Avec Mongoose, les objets imbriqués ne se comportent plus comme avant
```

Comment:
This comment is meaningless: what is “like before”? If this is only related to an API change, there is no sense in making this a comment when its contents are time-bound.